roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -38,6 +40,8 @@ class TransitionSpec extends FlatSpec with Matchers {
 
   val globalEnv = Seq.fill(669)(Ob.NIV).updated(668, stdOprnPlus)
 
+  val xferGlobalToReg_plus = OpXferGlobalToReg(r = 1, g = 668)
```

Comment:
Ok, I just wanted to it a bit clearer for a reader that there is used plus operation.

Reply:
Yes, definitely a good idea.
My first comment about going for something like `GlobalEnv.StdOprnPlus` was bad since we will only need it for our test cases. Therefore `indexStdOprnPlus` in `TransitionSpec.scala` probably makes more sense.